Dockerfile

```diff
@@ -1,42 +1,15 @@
-ARG CUDA_VERSION=11.4.3
-FROM nvidia/cuda:${CUDA_VERSION}-cudnn8-runtime-ubuntu20.04 as cuda_image
-FROM ubuntu:20.04
+FROM marqoai/marqo-base:15 as base_image
 VOLUME /var/lib/docker
 ARG TARGETPLATFORM
-# this is required for onnx to find cuda
-COPY --from=cuda_image /usr/local/cuda/ /usr/local/cuda/
 WORKDIR /app
-RUN apt-get update
-RUN apt-get install ca-certificates curl  gnupg lsof lsb-release jq -y
-RUN apt-get install apt-transport-https ca-certificates curl gnupg2 software-properties-common -y
-RUN apt-get update
-RUN add-apt-repository ppa:deadsnakes/ppa
-RUN apt-get update
-RUN apt-get install python3.8-distutils -y # python3-distutils
-RUN apt-get  install python3.8 python3-pip -y # pip is 276 MB!
-# opencv requirements
-RUN apt-get install ffmpeg libsm6 libxext6 -y
 
-# Punkt Tokenizer
-RUN apt-get install unzip -y
-RUN mkdir -p /root/nltk_data/tokenizers
-RUN curl https://raw.githubusercontent.com/nltk/nltk_data/gh-pages/packages/tokenizers/punkt.zip -o /root/nltk_data/tokenizers/punkt.zip
-RUN unzip /root/nltk_data/tokenizers/punkt.zip  -d /root/nltk_data/tokenizers/
-
-# TODO: up the RAM
-RUN echo Target platform is "$TARGETPLATFORM"
 COPY requirements.txt requirements.txt
 RUN pip3 install --no-cache-dir -r requirements.txt
+
 COPY scripts scripts
-RUN bash scripts/install_onnx_gpu_for_amd.sh
-RUN bash scripts/install_torch_amd.sh
-# redis installation for throttling
-RUN bash scripts/install_redis.sh
-# redis config lines
-RUN echo "echo never > /sys/kernel/mm/transparent_hugepage/enabled" >> /etc/rc.local
-RUN echo "save ''" | tee -a /etc/redis/redis.conf
-COPY dind_setup dind_setup
-RUN bash dind_setup/setup_dind.sh
+RUN bash scripts/dind_setup/setup_dind.sh
+
+FROM base_image
```

Comment:
I'm guessing this is superfluous and won't change the final image.

Reply:
If this is simply superfluous do you mind if we keep this as-is for now? The current image went through quite a long testing process, and I do vaguely recall that removing this caused some issues (though I could be misremembering)

Reply:
Fair enough. Yes I don't think it'll have any major effect (good or bad). Let's keep it for now then.